articles/zh/procedural-macros-in-rust.md

```diff
@@ -1799,8 +1802,8 @@ impl Parse for ParsedMap {
     fn parse(input: ParseStream) -> syn::Result<Self> {
         let mut entries = Vec::<ParsedMapEntry>::new();
 
-        // 检查输入是否为空（没有参数传递）。如果不是，则
-        // 报错，因为我们无法继续下去。
+        // 检查输入是否为空（没有参数传递）。
+        // 如果不是，则报错，因为我们无法继续下去。
         if input.is_empty() {
             panic!("至少要为一个空的 hashmap 指定一个类型");
```

Comment:
```suggestion
            panic!("At least a type must be specified for an empty hashmap");
```

Reply:
these are just prompts that will be printed, for chinese readers' better understanding i guess it's okay to translate them? wdyt

Reply:
Great. Let's translate them.